docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: '3.8'
+
+services:
+  postgres:
+    image: postgres:14-alpine
+    # comment out if you want to externally connect DB
+    # ports:
+    #   - 5432:5432
+    volumes:
+      - ./postgres-data:/var/lib/postgresql/data
+    environment:
+      - PGDATA=/var/lib/postgresql/data/pgdata
+      - POSTGRES_USER=skyvern
+      - POSTGRES_PASSWORD=skyvern
+      - POSTGRES_POSTGRES_DB=skyvern
+
+  skyvern:
+    image: public.ecr.aws/t6d4b5t4/skyvern:latest
+    restart: on-failure
+    # comment out if you want to externally call skyvern API
+    # ports:
+    #   - 8000:8000
+    volumes:
+      - ./artifacts:/data/artifacts
+      - ./videos:/data/videos
+      - ./har:/data/har
+      - ./.streamlit:/app/.streamlit
+    environment:
+      - DATABASE_STRING=postgresql+psycopg://skyvern:skyvern@postgres:5432/skyvern
+      - BROWSER_TYPE=chromium-headful
+      - ENABLE_OPENAI=true
+      - OPENAI_API_KEY=<your_openai_key>
+      # If you want to use other LLM provider, like azure and anthropic:
+      # - ENABLE_ANTHROPIC=true
+      # - ANTHROPIC_API_KEY=<your_anthropic_key>
+      # - ENABLE_AZURE=true
+      # - LLM_KEY=AZURE_OPENAI_GPT4V
+      # - AZURE_DEPLOYMENT=<your_azure_deployment>
+      # - AZURE_API_KEY=<your_azure_api_key>
+      # - AZURE_API_BASE=<your_azure_api_base>
+      # - AZURE_API_VERSION=<your_azure_api_version>
+    depends_on:
+      - postgres
```

Comment:
This wouldn't work well because this way it just waits for the container to be up. Instead, we should be waiting for postgres to be ready. 

This eventually works out but until skyvern is up, the container will restart multiple times since it'll die without a healthy DB.

Reply:
Great, I have noticed it already.
But I don't have any idea how to make skyvern check the DB health hook in docker-compose.
Any good idea?

Reply:
So I just let skyvern restart on failure. haha

Reply:
Yeah it works but if someone checks out the logs or runs without `-d` it could get confusing 😄 

Can you check this out and see if it works? I did something similar before, if this doesn't work we can hop on a call later?

https://stackoverflow.com/a/55835081

Reply:
It worked.